### PR TITLE
meeral added doc for IoT Scenario

### PR DESCRIPTION
**Pull request by msdpalam**
I have just added the doc for the scenario "Real time IoT data processing on Azure". Can you please pull and review and let me know your feedback?

Thanks.

Meer Alam (meeral)

### CODE REVIEW

File: docs/example-scenario/data/realtime-analytics-vehicle-iot.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+* Vehicle Maintenance Reminders and Alerting.
+* Location based services for the vehicle passengers (i.e. SOS)
+* Autonomous (Self-driving) vehicles.
+
+
+## Architecture
+
+![Screenshot](media/architecture-diagram-realtime-analytics-vehicle-data1.png)
+
+Big Data pipelines in Azure implements Lambda architecture. The solution falls into the speed layer (the layer that processes data in motion) of Lambda architecture. The data flows through the solution as follows:
```
Comment (MikeWasson) [inline docs/example-scenario/data/realtime-analytics-vehicle-iot.md]: If this only implements the speed layer (hot path) then by definition it's not a lambda architecture. You could call it kappa or just real-time stream processing.

Comment (msdpalam) [reply docs/example-scenario/data/realtime-analytics-vehicle-iot.md]: Strictly speaking I agree with you. I was trying to point to the fact that we can compare the implementation similar to speed layer in lambda architecture. I can elaborate on the fact that, once stored in the serving layer, the batch layer can follow from there, that would be a Lamda architecture context, however the doc then needs to be extended, in my opinion. Not sure if we want to extend further. May be will just keep as "real time big data analytic platform"? Thoughts?

Comment (MikeWasson) [reply docs/example-scenario/data/realtime-analytics-vehicle-iot.md]: Got it, I see what you're saying...  but I see a lot of confusion about lambda architecture, might be better just to call it real-time stream processing or equivalent.

Comment (msdpalam) [reply docs/example-scenario/data/realtime-analytics-vehicle-iot.md]: Made the changes and committed the update. Can you please review and let me know if this looks good now?
Thanks for your help reviewing the document

File: docs/example-scenario/data/realtime-analytics-vehicle-iot.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: Real time IoT data processing on Azure
```
Comment (adamboeglin) [inline docs/example-scenario/data/realtime-analytics-vehicle-iot.md]: How about something like "Ingestion and Processing of real-time automotive IOT data"?

Comment (msdpalam) [reply docs/example-scenario/data/realtime-analytics-vehicle-iot.md]: Sounds good to me, suits better here, thanks for the recommendation. I will change it and commit back. Please let me know if you have any other thoughts.

**Pull request merged**